src/platforms/node/guides/aws-lambda/index.mdx

```diff
@@ -16,37 +19,60 @@ $ npm install --save @sentry/node
 $ yarn add @sentry/node
 ```
 
-To set up Sentry error logging for a Lambda Function, build a wrapper:
+You can use the AWS Lambda integration for the Node SDK like this:
 
 ```javascript
-"use strict";
+const Sentry = require("@sentry/serverless");
+
+Sentry.init({
+  dsn:
+    "___PUBLIC_DSN___",
+});
+
+const myAsyncHandler = async (event, context, callback) => {
+  //Your handler code
+};
+
+exports.handler = Sentry.AWSLambda.wrapHandler(myAsyncHandler);
+```
+
+<!-- TODO-ADD-VERIFICATION-EXAMPLE -->
 
-const Sentry = require("@sentry/node");
+## Timeout Warning
 
+Sentry reports timeout warning when the function is within 500ms of it's execution time. You can turn off timeout warnings by setting `captureTimeoutWarning` to `false` during initiatlization. To change timeout warning limit, assign a numeric value (in ms) to `timeoutWarning`.
+
+```javascript {tabTitle:captumeTimeoutWarning}
 Sentry.init({
-  dsn: "___PUBLIC_DSN___",
+  dsn:
+    "___PUBLIC_DSN___",
+    captureTimeoutWarning: false
 });
+```
 
-function sentryHandler(lambdaHandler) {
-  return async event => {
-    try {
-      return await lambdaHandler(event);
-    } catch (e) {
-      Sentry.captureException(e);
-      await Sentry.flush(2000);
-      return e;
-    }
-  };
-}
-
-module.exports.hello = sentryHandler(async event => {
-  notExistFunction();
-  return event;
+```javascript {tabTitle:timeoutWarning}
+Sentry.init({
+  dsn:
+    "___PUBLIC_DSN___",
+    timeoutWarning: 50
 });
 ```
 
-You can obtain the DSN using your Sentry account from your organization's _Settings > Projects > Client Keys (DSN)_ in the Sentry web UI.
+The timeout warning is sent only if the "timeout" in the Lambda Function configuration is set to a value greater than one second.
```

Comment:
There's nothing stopping you from setting timeout to `1s` and still capture the warning if limit is small enough.